python/mxnet/module/base_module.py

```diff
@@ -670,3 +675,29 @@ def symbol(self):
         not even be associated with any symbols.
         """
         return self._symbol
+
+
+# pylint: disable=W0622
+class DataDesc(namedtuple('DataDesc', ['name', 'shape'])):
+    """Named data desc description contains name, shape, type and other extended attributes.
+    """
+    def __new__(cls, name, shape, type=float, layout='NCHW'):
+        ret = super(cls, DataDesc).__new__(cls, name, shape)
+        ret.type = type
```

Comment:
OK，I have changed to use mx_real_t. but CI still got errors related to network issues